src/codemagic/models/certificate_p12_exporter.py

```diff
@@ -42,17 +87,13 @@ def _get_export_path(cls, export_path: Optional[pathlib.Path]) -> pathlib.Path:
         with tempfile.NamedTemporaryFile(prefix="certificate", suffix=".p12", delete=False) as tf:
             return pathlib.Path(tf.name)
 
-    @classmethod
-    def _ensure_openssl(cls):
-        if shutil.which("openssl") is None:
-            raise IOError("OpenSSL executable is not present on system")
-
     def _run_openssl_command(self, command: Sequence[Union[str, pathlib.Path]]):
         process = None
         cli_app = self.get_current_cli_app()
         try:
             if cli_app:
-                process = cli_app.execute(command, [command[-1]])
+                obfuscate_patterns = [arg for arg in command if str(arg)[:5] == "pass:"]
```

Comment:
```suggestion
                obfuscate_patterns = [arg for arg in command if str(arg).startswith("pass:")]
```

Reply:
Good catch. I managed to mix up `str.startswith` and [`str.removeprefix`](https://docs.python.org/3.9/library/stdtypes.html#str.removeprefix). The latter cannot be used as it requires Python 3.9, but our lowest bound is 3.7.